### DISCUSSION TITLE
articles/nostalgia

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#180] [Comments]
# articles/nostalgia

上海美术电影制片厂和哔哩哔哩今年联合推出的《中国奇谭》在网络上收获了大量好评，一共八集，我花了几个周末追完，感觉都非常不错。印象最深的是前四集。第一集《小妖怪的夏天》最幽默，第二集《鹅鹅鹅》最具艺术气息，第三集《林林》最有故事性，而我最喜欢的也是勾起回忆的则是第四集《乡村巴士带走了王孩儿和神仙》。

https://hutusi.com/articles/nostalgia

**Comment by wangyunzi** [comment]
看到最后才觉得原来大家的农村生活都是一样的

**Reply by hutusi** [reply]
是呀，农村的生活都一样，农村的变化感觉也是一样。

**Comment by xibolun** [comment]
这文采！！！

**Reply by hutusi** [reply]
真实记录

**Comment by acechef** [comment]
写的真好

**Reply by hutusi** [reply]
谢谢鼓励~

**Comment by BYGD** [comment]
来看看老哥~~

**Reply by hutusi** [reply]
🤝

**Comment by syscn** [comment]
读完全文，有一种说不出来的感觉。很难受。

**Comment by longmenzhitong** [comment]
唯有真实最动人。👍
